examples/direct-upload/README.md

```diff
@@ -71,8 +71,28 @@ following into the CORS field
 (*it's basically saying that we are allowing GET,
  POST and PUT requests from any Allowed Origin with any Allowed Header*)  
 
++ Finally we need to add a policy to the bucket to make it public readable. Click
```

Comment:
@jackcarlisle might be worth clarifying that this makes the _files_ uploaded to the bucket viewable in browsers.